postgresql-s3-backup/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for progresql-s3-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cronjob:
+  # At 01:00 every day
+  schedule: ""
+  annotations: {}
+  concurrencyPolicy: ""
+  failedJobsHistoryLimit: ""
+  successfulJobsHistoryLimit: ""
+
+config: 
+  s3:
+    region: 'eu-west-2'
+    bucket: 'stackworx-k8-dev-database-backup'
+    #prefix: 'bucket'
+  #postgres:
+    #database: 'postgres'
+    #user: 'postgres'
+    #host: 'host port or name' 
```

Comment:
postgres:
  user: postgres
  database: postgres

These should be uncommented or they will be ignored